source/dbo/Stored Procedures/Power BI Reporting/uspUpdateRecordPostcode.sql

```diff
@@ -1,42 +1,44 @@
 ﻿/*
 	This procedure creates a cleaned postcode from the value supplied from the notification system (ETS or NTBS)
 	Provided the postcode matches the format and length of a UK postcode, this will insert a single space 3 characters from
-	the right hand side of the string
+	the right hand side of the string. Otherwise, it will leave the postcode value unchanged
 */
 
 CREATE PROCEDURE [dbo].[uspUpdateRecordPostcode] AS
 	SET NOCOUNT ON
 
 	BEGIN TRY
-		
 
-		IF object_id('tempdb.dbo.#RecordPostcodes','U') IS NOT NULL 
-		BEGIN	
+
+		IF object_id('tempdb.dbo.#RecordPostcodes','U') IS NOT NULL
+		BEGIN
 			DROP TABLE #RecordPostcodes
 		END
 
-		SELECT DISTINCT 
+		SELECT DISTINCT
 			PersonalDetailsId
 			,REPLACE(PostcodeToLookup, ' ', '')	AS CleanedPostcode
 		INTO #RecordPostcodes
 		FROM [dbo].[Record_PersonalDetails]
-  
-		DELETE from #RecordPostcodes 
-		WHERE LEN(CleanedPostcode)<5 OR LEN(CleanedPostcode) > 7 OR CleanedPostcode NOT LIKE '%[^0-9]%'
+
+		DELETE from #RecordPostcodes
+		WHERE CleanedPostcode IS NULL
+			OR LEN(CleanedPostcode)<5
+			OR LEN(CleanedPostcode) > 7
```

Comment:
The inconsistency in spacing here is hard to look at!

Reply:
Yep, fixed now!